minecraft_servers/warfaremc/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "server_name": "warfaremc",
+  "nice_name": "WarfareMC",
+  "direct_ip": "mc.warfaremc.eu",
+  "server_wildcards": [
+    "play.warfaremc.eu"
+  ],
+  "supported_languages": [
+    "cz",
+    "sk"
+  ],
+  "social": {
+    "web": "https://warfaremc.eu",
+    "web_shop": "https://shop.warfaremc.eu",
```

Comment:
https://store.warfaremc.eu/ would be the direct link ig.

Reply:
Fixed url, should be fine now.